### PR TITLE
feat: wallet management v2

### PR DESCRIPTION
**Pull request by chybisov**
Accidentally merged previous PR 😅

@Addminus please check comments from the previous one #67

### CODE REVIEW

**Comment by chybisov** [comment]
@Addminus I have the same issue with switching chains with the new implementation 

image

**Comment by chybisov** [comment]
Related https://github.com/MetaMask/metamask-extension/issues/18509

File: packages/widget/src/pages/SelectWalletPage/SelectWalletPage.tsx [hunk 1]
```diff
@@ -61,7 +67,24 @@ export const SelectWalletPage = () => {
...
+                alt={wallet.name}
+              >
+                {wallet.name[0]}
+              </Avatar>
+            </ListItemAvatar>
+            <ListItemText primary={wallet.name} />
+          </ListItemButton>
+        ))}
+
+        {notInstalledWallets.map((wallet: Wallet) => (
```
Comment (chybisov) [inline packages/widget/src/pages/SelectWalletPage/SelectWalletPage.tsx:87]: We can probably merge these arrays and map through one. 🙂

Comment (Addminus) [reply packages/widget/src/pages/SelectWalletPage/SelectWalletPage.tsx:87]: true

Comment (Addminus) [reply packages/widget/src/pages/SelectWalletPage/SelectWalletPage.tsx]: fixed

**Pull request merged**